golden_response/staging/get_stats/TotalCrimes.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
Is this based on some version of data or it's based on test data?

Reply:
This is actual data from current version.

Reply:
I c. Not sure how often it changes, otherwise, might be to craft some test data.

Reply:
This is indeed a prober golden test against the staging environment. This is to choose the less changed yearly updated data.

Reply:
How is this test different from the staging prober? Will we now have to maintain/update one more set of golden files?

If the intent is end-to-end test of mixer, should we do something less real with fakes?

Reply:
The staging prober and this test will share exact the same queries and golden response.

This one runs as presubmit check and enforce code change to update the golden files before sumission